mmf/modules/optimizers.py

```diff
@@ -1,7 +1,9 @@
 # Copyright (c) Facebook, Inc. and its affiliates.
 
+from torch.optim import Adam
 from transformers.optimization import AdamW
 
 from mmf.common.registry import registry
 
+registry.register_optimizer("adam")(Adam)
```

Comment:
This can be directly used. Why register?

Reply:
@vedanuj We can delete it since I also train the network using AdamW